<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gb_connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Připojení k MTB</string>
    </property>
    <widget class="QLineEdit" name="le_address">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>localhost</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_port">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>3841</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>adresa</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_connect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Připojit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_disconnect">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Odpojit</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Servo</string>
    </property>
    <widget class="QSpinBox" name="sb_servo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>6</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_module">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="value">
      <number>16</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_set_stop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ukončit nastavení</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>adresa</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>servo (1-6)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Pozice</string>
    </property>
    <widget class="QPushButton" name="pb_minus">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_plus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_plusplusplus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>+++</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_minusminusminus">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>---</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_pos_act">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00000</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_servopos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_servoposset">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Pozice</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_plusplus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>++</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_minusminus">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_io">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>221</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>Vstupy/Výstupy</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
